src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (newRobot.hasOwnProperty(value)) {
+      return null;
+    } else {
+      newRobot[value] = key;
+    }
+  }
```

Comment:
You shouldn't use ` else` after the return statement (the 4th point in the checklist).